Packs/Core/Playbooks/Playbook-Large_Upload.yml

```diff
@@ -0,0 +1,3868 @@
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: FindSimilarAlerts
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      filterQuery:
+        simple: alert.closeReason:"Resolved - False Positive" or alert.closeReason:"Resolved - Known Issue"
+      hoursBack:
+        simple: "168"
+      ignoreClosedIncidents:
+        simple: "no"
+      incidentFieldsAppliedCondition:
+        simple: AND
+      similarIncidentFields:
+        simple: alert.name,alert.initiatedby,alert.remoteport,alert.initiatorcmd,alert.osparentcmd,alert.remoteip,alert.remotehost,alert.cgocmd,alert.cgosha256,alert.osparentsha256,alert.username,alert.initiatorsha256,alert.cgoname,alert.osparentname,alert.appid,
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 25929536-ed1b-4848-810f-8b2023b1d8fb
+    type: condition
+    task:
+      id: 25929536-ed1b-4848-810f-8b2023b1d8fb
+      version: -1
+      name: Found Results?
+      description: Determine if previous false positive alerts have been detected with similar characteristics.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "6"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: isSimilarIncidentFound
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: dc4effe6-7a0d-45d2-8301-15c8d4234cd0
+    type: condition
+    task:
+      id: dc4effe6-7a0d-45d2-8301-15c8d4234cd0
+      version: -1
+      name: Investigation Should be Continued?
+      description: Determines whether an alert requires further investigation if similar previous false positive incidents were found.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
```

Comment:
the default is false means that the PB stops and does nothing - maybe we should close the alert?

Reply:
done. added the following tasks:

1. task number 56 - Close Alert - Resolve As a Duplicate Alert - closes the alert as duplicated if similarly closed alerts were found.
2. task number 57 - Close Alert - Resolve As False Positive - closes the alert as false positive if the verdict is benign or the verdict is suspicious and the user reviewed the alert and decided no remediation is needed.
3. task number 58 - Close Alert - Resolve As True Positive - closes the alert as true positive if the verdict is malicious or the verdict is suspicious and the user reviewed the alert and decided that remediation is needed.